docs/content/deploying-to-hippo.md

```diff
@@ -3,99 +3,85 @@ template = "main"
 date = "2022-05-20T00:22:56Z"
 [extra]
 url = "https://github.com/fermyon/spin/blob/main/docs/content/deploying-to-hippo.md"
-
 ---
 
-The Spin CLI offers a command to deploy a spin application to the Hippo platform.
+The Spin CLI offers a command to deploy a spin application to the Hippo
+platform.
 
 ## Deploying Spin applications to Hippo
 
-[Hippo](https://github.com/deislabs/hippo) is a Platform as a Service (PaaS) for running WebAssembly applications. Recently,
-Hippo has added support for scheduling WebAssembly modules on Nomad using Spin. The
-`spin deploy` command is an experimental command for deploying Spin applications to
-Hippo locally or in the cloud. This document provides guidance for how to get set up
-locally. Documentation on how to get set up to deploy to the cloud using `spin deploy` is
-coming soon.
+[Hippo](https://github.com/deislabs/hippo) is a Platform as a Service (PaaS)
+for running WebAssembly applications. Recently, Hippo has added support for
+scheduling WebAssembly modules on Nomad using Spin. The `spin deploy` command
+is an experimental command for deploying Spin applications to Hippo locally or
+in the cloud.
+
+This document provides guidance for how to get set up locally. Documentation on
+how to get set up to deploy to the cloud using `spin deploy` is coming soon.
 
 ### Pre-requisites
 
 - [Spin](https://github.com/fermyon/spin/releases)
-- [Hippo CLI latest](https://github.com/deislabs/hippo-cli)
+- [Hippo CLI latest](https://github.com/deislabs/hippo-cli/releases)
 - [Bindle v0.8.0](https://github.com/deislabs/bindle/releases/tag/v0.8.0)
 - [Nomad >= v1.2.6](https://www.nomadproject.io/)
 - [Consul >= v1.11.3](https://www.consul.io/)
 - [Vault >= v1.9.3](https://www.vaultproject.io/)
 - [Traefik >= 2.6.1](https://github.com/traefik/traefik/releases)
-- [Dotnet 6.0 CLI](https://dotnet.microsoft.com/en-us/download)
 
 ### Getting Set Up
 
-Start Nomad, Consul, and Traefik using the `run_servers.sh` script from the [nomad-local-demo repo](https://github.com/fermyon/nomad-local-demo).
+Start Nomad, Consul, and Traefik using the `run_servers.sh` script from the
+[nomad-local-demo repo](https://github.com/fermyon/nomad-local-demo).
```

Comment:
This can now change to point to fermyon/installer... Not sure if we want to link specifically to https://github.com/fermyon/installer/tree/main/local
```suggestion
[fermyon/installer repo](https://github.com/fermyon/installer).
```